src/chk/check.pb-c.c

```diff
@@ -0,0 +1,317 @@
+  { "CIC_POOL_NONEXIST_ON_MS", "CHK__CHECK_INCONSIST_CLASS__CIC_POOL_NONEXIST_ON_MS", 4 },
+  { "CIC_POOL_NONEXIST_ON_ENGINE", "CHK__CHECK_INCONSIST_CLASS__CIC_POOL_NONEXIST_ON_ENGINE", 5 },
+  { "CIC_POOL_BAD_SVCL", "CHK__CHECK_INCONSIST_CLASS__CIC_POOL_BAD_SVCL", 6 },
+  { "CIC_POOL_BAD_LABEL", "CHK__CHECK_INCONSIST_CLASS__CIC_POOL_BAD_LABEL", 7 },
+  { "CIC_ENGINE_NONEXIST_IN_MAP", "CHK__CHECK_INCONSIST_CLASS__CIC_ENGINE_NONEXIST_IN_MAP", 8 },
+  { "CIC_ENGINE_DOWN_IN_MAP", "CHK__CHECK_INCONSIST_CLASS__CIC_ENGINE_DOWN_IN_MAP", 9 },
+  { "CIC_ENGINE_HAS_NO_STORAGE", "CHK__CHECK_INCONSIST_CLASS__CIC_ENGINE_HAS_NO_STORAGE", 10 },
+  { "CIC_CONT_NONEXIST_ON_PS", "CHK__CHECK_INCONSIST_CLASS__CIC_CONT_NONEXIST_ON_PS", 11 },
+  { "CIC_CONT_BAD_LABEL", "CHK__CHECK_INCONSIST_CLASS__CIC_CONT_BAD_LABEL", 12 },
+  { "CIC_DTX_CORRUPTED", "CHK__CHECK_INCONSIST_CLASS__CIC_DTX_CORRUPTED", 13 },
+  { "CIC_DTX_ORPHAN", "CHK__CHECK_INCONSIST_CLASS__CIC_DTX_ORPHAN", 14 },
+  { "CIC_CSUM_LOST", "CHK__CHECK_INCONSIST_CLASS__CIC_CSUM_LOST", 15 },
+  { "CIC_CSUM_FAILURE", "CHK__CHECK_INCONSIST_CLASS__CIC_CSUM_FAILURE", 16 },
+  { "CIC_OBJ_LOST_REP", "CHK__CHECK_INCONSIST_CLASS__CIC_OBJ_LOST_REP", 17 },
+  { "CIC_OBJ_LOST_EC_SHARD", "CHK__CHECK_INCONSIST_CLASS__CIC_OBJ_LOST_EC_SHARD", 18 },
+  { "CIC_OBJ_LOST_EC_DATA", "CHK__CHECK_INCONSIST_CLASS__CIC_OBJ_LOST_EC_DATA", 19 },
+  { "CIC_OBJ_DATA_INCONSIST", "CHK__CHECK_INCONSIST_CLASS__CIC_OBJ_DATA_INCONSIST", 20 },
+  { "CIC_UNKNOWN", "CHK__CHECK_INCONSIST_CLASS__CIC_UNKNOWN", 100 },
+};
+static const ProtobufCIntRange chk__check_inconsist_class__value_ranges[] = {
+{0, 0},{100, 21},{0, 22}
+};
+static const ProtobufCEnumValueIndex chk__check_inconsist_class__enum_values_by_name[22] =
+{
+  { "CIC_CONT_BAD_LABEL", 12 },
+  { "CIC_CONT_NONEXIST_ON_PS", 11 },
+  { "CIC_CSUM_FAILURE", 16 },
+  { "CIC_CSUM_LOST", 15 },
+  { "CIC_DTX_CORRUPTED", 13 },
+  { "CIC_DTX_ORPHAN", 14 },
+  { "CIC_ENGINE_DOWN_IN_MAP", 9 },
+  { "CIC_ENGINE_HAS_NO_STORAGE", 10 },
+  { "CIC_ENGINE_NONEXIST_IN_MAP", 8 },
+  { "CIC_NONE", 0 },
+  { "CIC_OBJ_DATA_INCONSIST", 20 },
+  { "CIC_OBJ_LOST_EC_DATA", 19 },
+  { "CIC_OBJ_LOST_EC_SHARD", 18 },
+  { "CIC_OBJ_LOST_REP", 17 },
+  { "CIC_POOL_BAD_LABEL", 7 },
+  { "CIC_POOL_BAD_SVCL", 6 },
+  { "CIC_POOL_LESS_SVC_WITHOUT_QUORUM", 2 },
+  { "CIC_POOL_LESS_SVC_WITH_QUORUM", 1 },
+  { "CIC_POOL_MORE_SVC", 3 },
+  { "CIC_POOL_NONEXIST_ON_ENGINE", 5 },
+  { "CIC_POOL_NONEXIST_ON_MS", 4 },
+  { "CIC_UNKNOWN", 21 },
+};
+const ProtobufCEnumDescriptor chk__check_inconsist_class__descriptor =
+{
+  PROTOBUF_C__ENUM_DESCRIPTOR_MAGIC,
+  "chk.CheckInconsistClass",
+  "CheckInconsistClass",
+  "Chk__CheckInconsistClass",
+  "chk",
+  22,
+  chk__check_inconsist_class__enum_values_by_number,
+  22,
+  chk__check_inconsist_class__enum_values_by_name,
+  2,
+  chk__check_inconsist_class__value_ranges,
+  NULL,NULL,NULL,NULL   /* reserved[1234] */
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)